dlt-connector/src/interactions/backendToDb/community/HomeCommunity.role.ts

```diff
@@ -36,12 +38,14 @@ export class HomeCommunityRole extends CommunityRole {
 
   public async store(): Promise<Community> {
     try {
-      return await getDataSource().transaction(async (transactionalEntityManager) => {
+      const community = await getDataSource().transaction(async (transactionalEntityManager) => {
         const community = await transactionalEntityManager.save(this.self)
         await transactionalEntityManager.save(this.transactionRecipe)
         logger.debug('store home community', new CommunityLoggingView(community))
         return community
       })
+      InterruptiveSleepManager.getInstance().interrupt(TRANSMIT_TO_IOTA_INTERRUPTIVE_SLEEP_KEY)
```

Comment:
in tasks/transmitToIota.ts:
```ts
await InterruptiveSleepManager.getInstance().sleep(
        TRANSMIT_TO_IOTA_INTERRUPTIVE_SLEEP_KEY,
        1000,
      )
```
It will interrupt the sleep of the transmitToIota Loop.
It is modeled after conditions used in multithreading in java and c++
The Loop in transmitToIota:
- check for new transactions for sending to iota
- sleep 10000 ms 
  - if  InterruptiveSleepManager.getInstance().interrupt(TRANSMIT_TO_IOTA_INTERRUPTIVE_SLEEP_KEY) is called, the sleep will be interrupted 
- repeat